pkg/plugin/installer/installer.go

```diff
@@ -17,6 +17,7 @@ package installer
 
 import (
 	"fmt"
+	"helm.sh/helm/v3/pkg/plugin"
```

Comment:
Move this import below `github.com/pkg/errors` and separate it with a space. This is the format usually used in Helm Go files.